tablex.lua

```diff
@@ -537,4 +537,28 @@ function tablex.ripairs(t)
 	return _ripairs_iter, t, #t + 1
 end
 
+-- works like pairs, but returns sorted table
+function tablex.spairs(t, fn)
+	local keys = {}
+	for k in pairs(t) do
+		tablex.push(keys, k)
+	end
+
+	if fn then
+		table.sort(keys, function(a,b) return fn(t, a, b) end)
```

Comment:
is there a reason the signature for the function provided isn't just `f(a, b)` - seems like it'd be expected?

Reply:
Sorry, I don't understand what do I need to change. Could you please write in more detail?